backend/src/test/java/backend/mulkkam/intake/service/IntakeAmountServiceIntegrationTest.java

```diff
@@ -96,6 +103,37 @@ void error_memberIsNotExisted() {
                     () -> intakeAmountService.modifyTarget(intakeTargetAmountModifyRequest, Long.MAX_VALUE));
             assertThat(exception.getErrorCode()).isEqualTo(NotFoundErrorCode.NOT_FOUND_MEMBER);
         }
+
+        @DisplayName("오늘의 기록이 있다면 금일 목표 음용량도 변경한다")
+        @Test
+        void success_changeTodayTargetAmount() {
+            // given
+            Member member = MemberFixtureBuilder
+                    .builder()
+                    .targetAmount(new Amount(1500))
+                    .build();
+            memberRepository.save(member);
+            IntakeHistory intakeHistory = IntakeHistoryFixtureBuilder
+                    .withMember(member)
+                    .date(LocalDate.now())
+                    .targetIntakeAmount(new Amount(1500))
+                    .build();
+
+            intakeHistoryRepository.save(intakeHistory);
+
+            IntakeTargetAmountModifyRequest intakeTargetAmountModifyRequest = new IntakeTargetAmountModifyRequest(1000);
```

Comment:
```suggestion
            int newTargetIntakeAmount = 1_000;
            IntakeTargetAmountModifyRequest intakeTargetAmountModifyRequest = new IntakeTargetAmountModifyRequest(newTargetIntakeAmount);
```
이런식으로 해서 아래 검증문에서도 같은 변수로 쓰는 거 어때유~~?

Reply:
해당 부분 관련 칼리와 싸우기 약속

Reply:
맞다이다